apps/ui/src/components/Ui/WrapperInput.vue

```diff
@@ -1,18 +1,41 @@
 <script setup lang="ts">
-const props = defineProps<{
-  definition: any;
-  error?: string;
-  dirty?: boolean;
-}>();
+const props = withDefaults(
+  defineProps<{
+    definition: any;
+    inputValueLength?: number;
```

Comment:
Looks like it's called `input-length` in `InputString`, not sure which one you wanted to use.